app/scripts/routes.js

```diff
@@ -16,6 +16,10 @@ define([], function() {
                 templateUrl: 'views/students/index.html',
                 controller: 'StudentsIndexCtrl',
                 relativePath: '/students'
+            },
+            '/students/:docket/update': {
```

Comment:
It should be `/students/:docket/edit`.
`update` is the POST action that will be send to the server.

Why have you changed this from [the doc](https://docs.google.com/spreadsheets/d/1G42Rt0BrmhF9b7W_X0gIWFOwk2b41NUEaKYgSSbCOTE/edit#gid=0)? Please fix it also there

Also, rename the controller to be `StudentsEditCtrl`